tomviz/MainWindow.cxx

```diff
@@ -109,6 +109,15 @@ namespace tomviz {
 MainWindow::MainWindow(QWidget* parent, Qt::WindowFlags flags)
   : QMainWindow(parent, flags), m_ui(new Ui::MainWindow)
 {
+
+  // Override the default setting for showing full messages. This needs to be
+  // done prior to calling m_ui-setupUi(this)
+  pqSettings* qt_settings = pqApplicationCore::instance()->settings();
+  if (!qt_settings->contains("pqOutputWidget.ShowFullMessages")) {
+    qt_settings->setValue("pqOutputWidget.ShowFullMessages", true);
+    std::cout << "ShowFullMessages true" << std::endl;
```

Comment:
I don't think we should keep the output, the result should be obvious (I guess you wanted it for debugging)